app/config/errors.php

```diff
@@ -549,7 +549,7 @@
     ],
     Exception::ATTRIBUTE_ALREADY_EXISTS => [
         'name' => Exception::ATTRIBUTE_ALREADY_EXISTS,
-        'description' => 'Attribute with the requested ID already exists. Try again with a different ID or use "unique()" to generate a unique ID.',
+        'description' => 'Attribute with the requested ID already exists. Attribute IDs must be unique, try again with a different ID.',
```

Comment:
We call them keys rather than IDs so we should probably be consistent with that here.